workflows/package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "@filbeam/workflows",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Shared workflows for FilBeam workers",
+  "author": "Space Meridian <filbeam@meridian.space>",
+  "type": "module",
+  "exports": {
+    ".": "./lib/transaction-monitor-workflow.js"
+  },
+  "main": "lib/transaction-monitor-workflow.js",
```

Why do we need the `exports` block when we already have the `main` entry point specified? Would the following work?


```suggestion
  "main": "lib/transaction-monitor-workflow.js",
```